babel.config.js

```diff
@@ -2,5 +2,6 @@ module.exports = function(api) {
   api.cache(true);
   return {
     presets: ['babel-preset-expo'],
+    plugins: ["nativewind/babel"],
```

Comment:
We should use only 1 type of quotes, so it will be better if you change "nativewind/babel" to 'nativewind/babel'.